src/components/LinkButton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { routeTo } from 'redux-router-kit';
+import Link from '../components/LinkButton.scss';
```

Comment:
This was part of an idea where a link will have its own separate component, in reality the file shouldn't exist here